pkg/report/platform/types.go

```diff
@@ -3,16 +3,15 @@ package platform
 import (
 	"bytes"
 	"encoding/binary"
-	"encoding/hex"
 	"fmt"
 
-	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/ocr3/types"
+	"github.com/smartcontractkit/chainlink-evm/pkg/report/datafeeds"
 )
 
 // Report represents an OCR3 report with metadata and data
 // version | workflow_execution_id | timestamp | don_id | config_version | ... | data
 type Report struct {
```

Comment:
You changed this general (common) type to make it fit EVM, but common and Aptos used this type.
So are we now we're going to refactor Aptos, to use this new type introduced by EVM work?

Reply:
Yes Aptos will have to import from here until we generalize the type

Reply:
I would have preferred the other way around as a more cleaner approach, but ok now.

Either way we'll need to follow up here.